tests/test1.nims

```diff
@@ -0,0 +1 @@
+switch("path", "$projectDir/../src")
```

Comment:
Not a change request, just curious what this file is for.

Reply:
It's automatically generated by nimble.  It allows for unit testing.